components/molecules/Form/Form.js

```diff
@@ -24,22 +24,29 @@ export default function Form({
   validationSchema,
   onSubmit
 }) {
+  let formattedChildren = children
+  if ('function' !== typeof children) {
+    formattedChildren = () => children
+  }
+
   return (
     <Formik
       initialValues={formDefaults}
       validationSchema={validationSchema}
       onSubmit={onSubmit}
     >
-      <FormikForm id={id} className={cn(styles.form, className)}>
-        {children}
-        <button type="submit">Submit</button>
-      </FormikForm>
+      {(formikProps) => (
+        <FormikForm id={id} className={cn(styles.form, className)}>
+          {formattedChildren(formikProps)}
+          <button type="submit">Submit</button>
+        </FormikForm>
+      )}
```

Comment:
@oddevan Why the IIFEE here? Could you do:

```suggestion
      {!!formikProps && (
        <FormikForm id={id} className={cn(styles.form, className)}>
          {formattedChildren(formikProps)}
          <button type="submit">Submit</button>
        </FormikForm>
      )}
```

Reply:
The render function is so that `formikProps` can be passed from the `Formik` component into the `formattedChildren`. See https://formik.org/docs/api/formik#example